tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 minversion = 2.1
-envlist = py{35,27},functional,pep8,pip-missing-reqs
```

Comment:
Don't we need to test on python 3 anymore? I assume we want to go on a later Openstack version soon, which only works on python3

Reply:
Correct, there is another branch and pull request for Ussuri with python 3.6.9 (changes from both branches are in sync)
Here we use only python 2.7 due to some breaking changes.